.github/workflows/tests.yml

```diff
@@ -14,10 +14,7 @@ jobs:
     strategy:
       matrix:
         os: [macos-latest, windows-latest, ubuntu-20.04]
-        python-version: ['3.5', '3.6', '3.7', '3.8', '3.9', '3.10', '3.11']
-        exclude:
-          - os: windows-latest
-            python-version: 3.5
+        python-version: ['3.6', '3.7', '3.8', '3.9', '3.10', '3.11']
```

Comment:
Python 3.12 released recently, btw. But it may also be worth doing that in a different PR because they finally killed some deprecated features of `setup.py` and I don't know if we're impacted by any of them.

Reply:
(So, maybe add `'3.12'` to this PR, but if it fails, roll it back and we dig into it in a follow-up PR?)

Reply:
I'm anticipating some work to figure out the changes with 3.12, which will also involve updating the wheels matrix. I'd rather just do all the 3.12 work together in a separate PR.